src/js/api/config.js

```diff
@@ -161,11 +161,21 @@ define([
             config.playlist = configPlaylist.playlist;
         }
 
+        if (_.size(config.playlist)) {
+            config.playlist[0].preload = normalizePreload(config);
+        }
+
         config.qualityLabels = config.qualityLabels || config.hlslabels;
 
         return config;
     };
 
+    function normalizePreload(config) {
+        const preloadValues = ['none', 'meta', 'auto'];
+        const preload = config.playlist[0].preload || config.preload;
+
+        return _.contains(preloadValues, preload) ? preload : 'meta';
+    }
```

Comment:
@hghazzi  why are we checking that preload is contained in preloadValues ? Is this to prevent developers from typing a random preload value ?